### PR TITLE
pages/legal/licences.md: Add EPL-2.0 and EUPL-2.0

### PR DESCRIPTION
**Pull request by bzg**
A revision of the D. 323-2-1 article from the CRPA (code des relations
entre le public et l'administration) has been approved on Nov. 4th by
the "Conseil national d'évaluation des normes".

This change can be merged and released in production as soon as it is
made publicly official.

### CODE REVIEW

**Review by abulte — CHANGES_REQUESTED** [review]
Thanks, please ping us when it's official, it will be live almost instantly (I'm requesting change in this review to avoid accidental merge).

**Comment by bzg** [comment]
Le décret est paru : https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000044401895

**Comment by bzg** [comment]
Cc @maudetes

File: pages/legal/licences.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ La deuxième partie de cette page dresse la liste des licences spéciales homolo
 | [GNU Lesser General Public License v3.0 or later](https://spdx.org/licenses/LGPL-3.0-or-later.html#licenseText) | `LGPL-3.0-or-later` |
 | [GNU Affero General Public License v3.0 or later](https://spdx.org/licenses/AGPL-3.0-or-later.html#licenseText) | `AGPL-3.0-or-later` |
 | [Mozilla Public License 2.0](https://spdx.org/licenses/MPL-2.0.html#licenseText)                                | `MPL-2.0`           |
+| [Eclipse Public License 2.0](https://spdx.org/licenses/EPL-2.0.html#licenseText)                                | `EPL-2.0`           |
+| [European Union Public License 2.0](https://spdx.org/licenses/EUPL-1.2.html#licenseText)                        | `EUPL-2.0`          |
```
Comment (RLesur) [inline pages/legal/licences.md:50]: It seems there is a typo here.  
AFAIK, the latest version of the EUPL is 1.2. The url is correct but the text indicates 2.0.
I will open a PR.

Comment (bzg) [reply pages/legal/licences.md:50]: Thanks for spotting and reporting this, I just opened a PR.

**Pull request merged**